handbook/recruitment/overview.md

```diff
@@ -1,13 +1,13 @@
 # Recruitment Overview
 
-This document provides a overview of why and how we do recruitment at [UBC Launch Pad](https://ubclaunchpad.com). Our interview processes are documented in the [Leads repository](https://github.com/ubclaunchpad/leads).
-
-::: warning
-This page is a work in progress!
-:::
+This document provides a overview of why and how we do recruitment at [UBC Launch Pad](https://ubclaunchpad.com).
 
 ## Goals
 
+We have an application and interview process to join Launch Pad because software engineering is inherently a collaborative process — one that requires every contributor to work well with each other. We want to make sure that every member is commited and has the support required to do so, regardless what their skill level is. For the most part, this support comes from our [leads](/handbook/manifesto.md#project-teams), who also have to put in a great deal of effort to manage teams, plan projects, provide technical guidance, organize events, liaison with sponsors, and more. Given the small number of people at Launch Pad who have the background, time, and willingness to do all this, we sadly can’t accept and adequately support everyone who applies.
```

Comment:
```suggestion
We have an application and interview process to join Launch Pad because software engineering is inherently a collaborative process — one that requires every contributor to work well with each other. We want to make sure that every member is commited and has the support required to do so, regardless of what their skill level is. For the most part, this support comes from our [leads](/handbook/manifesto.md#project-teams), who also have to put in a great deal of effort to manage teams, plan projects, provide technical guidance, organize events, liaison with sponsors, and more. Given the small number of people at Launch Pad who have the background, time, and willingness to do all this, we sadly can’t accept and adequately support everyone who applies.
```